cmd/rigs/cmd/publish.go

```diff
@@ -20,9 +20,9 @@ import (
 )
 
 var (
-	_db        *sql.DB
-	_ethClient *ethclient.Client
+	_db *sql.DB
 
+	ethClient    *ethclient.Client
```

Comment:
All the changes in here are to make sure we always create an eth client we can then use to query for txn information, including cost.